aries-site/src/data/structures/components.js

```diff
@@ -850,6 +850,7 @@ export const components = [
   },
   {
     name: 'Menu',
+    accessibility: 'Failed WCAG 2.2 AA',
```

Comment:
Is this showing with the "green" outline because the [code from this PR](https://github.com/grommet/hpe-design-system/pull/4649/files) hasn't been merged?

Reply:
yes that is correct we were planning on merging button first then this will show the correct color